lighthouse/helpers/plates.py

```diff
@@ -215,8 +221,68 @@ def row_to_dict(row):
     return obj
 
 
+def request_with_retries(request_func: Callable, response_func: Callable[[Any], T], max_retries=3) -> T:
```

Comment:
As discussed on Slack, the retry was more of a disaster handler.  In reality, if this is actually needed, it would be a surprise.  I don't think I can define a better strategy than this one at the moment.